Classes/Emogrifier.php

```diff
@@ -397,6 +420,57 @@ public function removeUnprocessableHtmlTag($tagName)
     }
 
     /**
+     * Normalizes the value of the "style" attribute and saves it.
+     *
+     * @param \DOMElement $node
+     *
+     * @return void
+     */
+    private function saveOriginalStyle(\DOMElement $node)
```

Comment:
Yeah, you are correct. Your suggested name better reflects the purpose of the method. I changed this in my new commit.